app/models/project.rb

```diff
@@ -7,6 +7,7 @@ class Project < ApplicationRecord
   has_many :project_members, dependent: :destroy
   has_many :users, through: :project_members
   has_many :rules, dependent: :destroy
+  has_one :project_metadata, dependent: :destroy
 
```

Comment:
When controllers are implemented for managing projects we will probably want `accepts_nested_attributes_for :project_metadata` on the Project model

https://api.rubyonrails.org/classes/ActiveRecord/NestedAttributes/ClassMethods.html

Reply:
```suggestion
accepts_nested_attributes_for :project_metadata
```